pr-data.csv

```diff
@@ -1395,7 +1395,7 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndRemoveClosure,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndRemoveClosure_failureBranch,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndReplaceClosureIfExistsFalse_entryExists,ID,,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndReplaceClosureIfExistsFalse_entryNotExists,ID,,,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndReplaceClosureIfExistsFalse_entryNotExists,ID,MovedOrRenamed,,https://github.com/apache/ignite-3/commit/f370441e239c70b6ef371b819232533c028f1f4e
```

Comment:
The abstract test class where this test was moved seems to be extended with JUnit 5 annotation https://junit.org/junit5/docs/5.0.3/api/org/junit/jupiter/api/extension/ExtendWith.html (see https://github.com/apache/ignite-3/blob/56c217334884aac2ff4da6ca8023fa0d20a0028a/modules/storage-api/src/test/java/org/apache/ignite/internal/storage/AbstractPartitionStorageTest.java#L63).

Can you run the entire test suite for this module (just `mvn nondex:nondex`, without `-Dtest=...`) to see what tests get reported as flaky?

Reply:
Thanks for the comment. I ran the nondex tool against the whole modules/api. I do not see any of the above cases failing. Please look into 
fa22-cs527-009.cs.illinois.edu:/home/kf14/ignite-3/modules/storage-api/.nondex/eCteOUpebXtV+FA7rkxhRbz8MC3Y6+EC9ikZ6fBrU= if needed. 

For other flaky tests within this modules/storage-api, there are 2 assertion error test cases in ConcurrentHashMapMvTableStorageTest (both written in AbstractMvTableStorageTest.java and not included in pr-data.csv), 5 assertion error test cases in DataStorageManagerTest (Already included in pr-data.csv). Also, there are 34 test error ("Failed to resolve parameter") in TestSortedIndexStorageTest. All test case failures are all included in the link above.

Reply:
You need not list NetID publicly on GitHub; you can send private posts on Campuswire. For runs of NonDex, it's better to get a log, e.g., `mvn nondex:nondex |& tee log`? For new flaky tests that you see, you can add them later. Let me accept the current PR now.